crpc/generate.sh

```diff
@@ -5,39 +5,3 @@ protoc -I/usr/local/include -I. \
        -I$GOPATH/src \
```

Comment:
Please rename commit in "crpc: remove unrequried js and ts generation"

Reply:
fixed